docs/evm/Build/Guides/Deploy Solidity Contract/deploy-solidity-contract-with-hardhat.md

```diff
@@ -5,21 +5,235 @@ sidebar_label: Using Hardhat
 sidebar_position: 1
 ---
 
-# Deploying a Smart Contract using Hardhat
-
-This section is dedicated to guiding you through deploying smart contracts on the FlowEVM network using Hardhat. However, detailed documentation and tutorials on this topic are currently under development and will be available soon.
+# Using Hardhat
 
 Hardhat is a comprehensive Ethereum development tool that simplifies the process of deploying, testing, and debugging smart contracts. It's designed to offer developers a seamless experience when working with Solidity contracts.
 
+
+## Prerequisites
+
+### Software
+
+Node v18+, can be downloaded [Here](https://nodejs.org/en/download)
+
+Getting Started with Hardhat. [More Information](https://hardhat.org/tutorial/creating-a-new-hardhat-project). These directions demonstrate using npm commands to create a project and add its dependencies.
+
+```shell
+mkdir hardhat-example
+cd hardhat-example
+
+npm init
+
+npm install --save-dev hardhat
+
+npx hardhat init
+```
+
+
+### Fund Your Wallet
+
+Navigate to the [Crescendo PreviewNet Faucet](https://crescendo-faucet.onflow.org/fund-account). Paste in your wallet address to get receive $FLOW. Thi will allow your wallet address to deploy smart contracts. 
```

Comment:
```suggestion
Navigate to the [Crescendo PreviewNet Faucet](https://crescendo-faucet.onflow.org/fund-account). Paste in your wallet address to get receive $FLOW. This will allow your wallet address to deploy smart contracts and run transactions. 
```